File: 1351-count-negative-numbers-in-a-sorted-matrix/1351-count-negative-numbers-in-a-sorted-matrix.js
/**
 * @param {number[][]} grid
 * @return {number}
 */
function countNegatives(grid) {
  let totalCount = 0
    for(let i = 0; i < grid.length; i++){
       totalCount += checkNum(grid[i])
      }
      return totalCount;
};


function checkNum(arr){
    let count = 0
    for(let i = 0; i < arr.length; i++){
        if(arr[i] < 0) count++
    }
    return count;
}

File: 1351-count-negative-numbers-in-a-sorted-matrix/1351-count-negative-numbers-in-a-sorted-matrix.py
def checkNum(arr):
        count = 0
        for i in range(len(arr)):
            if(arr[i] < 0):
                count += 1
        return count 
        
class Solution:
    def countNegatives(self, grid: List[List[int]]) -> int:
        totalcount = 0
        for i in range(len(grid)):
            totalcount += checkNum(grid[i])
        return totalcount
